### PR TITLE
Enhance test covarege of MTOM with PAYLOAD data format

### PR DESCRIPTION
**Pull request by llowinge**

### CODE REVIEW

**Comment by llowinge** [comment]
@ppalaga Can you check it, thank you

**Review by ppalaga — APPROVED** [review]
Looks good, thanks @llowinge, one minor suggestion inline.

**Review by aldettinger — APPROVED** [review]

File: integration-test-groups/cxf-soap/cxf-soap-mtom/src/main/java/org/apache/camel/quarkus/component/cxf/soap/mtom/it/CxfSoapMtomResource.java [hunk 1]
```diff
@@ -56,22 +74,30 @@ public Response upload(@QueryParam("imageName") String imageName, @QueryParam("m
...
+            exchange.setPattern(ExchangePattern.InOut);
+            exchange.getIn().setBody(imageName);
+            exchange.getIn().setHeaders(headers);
+        });
+        Object response = null;
+        if ("PAYLOAD".equals(endpointDataFormat)) {
+            response = result.getMessage(AttachmentMessage.class).getAttachment(imageName).getContent();
+        } else {
+            response = result.getMessage().getBody(ImageFile.class).getContent();
+        }
```
[the remark this answers is not in the document]
Comment (llowinge) [reply integration-test-groups/cxf-soap/cxf-soap-mtom/src/main/java/org/apache/camel/quarkus/component/cxf/soap/mtom/it/CxfSoapMtomResource.java]: Fixed, but i had to do it bit differently. Thanks

**Review by ppalaga — APPROVED** [review]
Thanks @llowinge !

**Pull request merged**